ur_robot_driver/test/robot_driver.py

```diff
@@ -246,13 +139,8 @@ def io_msg_cb(msg):
         # Set pin 0 to 1.0
         test_pin = 0
 
-        set_io_req = SetIO.Request()
-        set_io_req.fun = 1
-        set_io_req.pin = test_pin
-        set_io_req.state = 1.0
-
-        self.node.get_logger().info(f"Setting pin {test_pin} to {set_io_req.state}")
-        self.call_service("/io_and_status_controller/set_io", set_io_req)
+        logging.info("Setting pin %d to 1.0", test_pin)
```

Comment:
Is there a reason you didn't use the `f"..."` notation? From my understanding that is "the recommended" way to format strings in Python nowadays.

Reply:
My understanding was that logging specifically still recommends the '%' syntax as it can avoid transforming the arguments to strings if the log output is not actually used. Pylint has warnings for this ([logging-fstring-interpolation](https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html)) which is why i stuck to this syntax for now, but i am open to changing this.